tests/Runtime/PhpFpmTest.php

```diff
@@ -144,11 +144,57 @@ public function test request with arrays in query string()
                 ],
             ],
             '$_SERVER' => [
-                'REQUEST_URI' => '/?vars%5Bval1%5D=foo&vars%5Bval2%5D%5B%5D=bar',
+                'REQUEST_URI' => '/?vars%5Bval1%5D=foo&vars%5Bval2%5D%5B0%5D=bar',
```

Comment:
Thank you, could you explain why this existing test has been changed?

Reply:
This is a default behavior of `\mb_parse_str` function.

Reply:
Thanks, but is it a regression?

Is this change valid? What do the extra `0` mean here?

Reply:
I don't see any threats. `foo[]=bar` is equal to `foo[0]=bar` and `foo[]=bar&foo[]=baz` to `foo[0]=bar&foo[1]=baz`.

Reply:
Ah OK I see, this is the array key. Thank you for the detail.

I need to look more into this and run some extra tests. Then we can merge this.

Thank you!